client/state/login/actions/create-social-user-failed.js

```diff
@@ -1,10 +1,23 @@
 import { SOCIAL_CREATE_ACCOUNT_REQUEST_FAILURE } from 'calypso/state/action-types';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import { getErrorFromWPCOMError } from 'calypso/state/login/utils';
 
 import 'calypso/state/login/init';
 
-export const createSocialUserFailed = ( socialInfo, error ) => ( {
-	type: SOCIAL_CREATE_ACCOUNT_REQUEST_FAILURE,
-	authInfo: socialInfo,
-	error: error.field ? error : getErrorFromWPCOMError( error ),
-} );
+export const createSocialUserFailed = ( socialInfo, error, startingPoint ) => ( dispatch ) => {
+	const wpcomError = error.field ? error : getErrorFromWPCOMError( error );
+
+	dispatch( {
+		type: SOCIAL_CREATE_ACCOUNT_REQUEST_FAILURE,
+		authInfo: socialInfo,
+		error: wpcomError,
+	} );
+
+	dispatch(
+		recordTracksEvent( 'calypso_social_auth_attempt', {
```

Comment:
@vindl please confirm this is the desired name of the event so @danielbachhuber can validate and register the event on Tracks. Thanks!

Reply:
Given that all of the other similar tracks events are prefixed with either `calypso_login_social_*` or `calypso_signup_social_*`, should we make the same differentiation here? (assuming that `startingPoint` will be limited to these values)

```suggestion
		recordTracksEvent( 'calypso_${startingPoint}_social_auth_attempt', {
```

That should allow us to differentiate how many of these attempts are happening in login flow vs signup flow.

Reply:
Per our huddle and points raised in https://github.com/Automattic/wp-calypso/pull/65865#discussion_r930295997, we're keeping the name as-is.